controllers/uaa.go

```diff
@@ -49,3 +98,476 @@ func (c *UAAContext) UaaInfo(rw web.ResponseWriter, req *web.Request) {
 		rw.Write([]byte("{\"status\": \"Bad request\", \"error_description\": \"Missing valid guid.\"}"))
 	}
 }
+
+// InviteEmail provides inviteTextTpl and inviteEmailTpl the value to access invite_url query
+type InviteEmail struct {
+	URL string
+}
+
+const inviteTextTpl = `
+You have been invited to join cloud.gov!
+Claim your account below.
+cloud.gov is a service by 18F that helps federal teams create and deliver quality digital services securely hosted in the cloud.
+
+Accept the invitation - Accept your invite to continue the registration process. You can also copy the URL below and paste it into your browser's address bar:
+{{.URL}}
+
+Accept your invitation
+Read the documentation - After you register and log in, review the acceptable uses and rules of behavior.
+
+Then set up your cloud.gov access and get started.
+
+If you run into problems or have any questions, please email us at cloud-gov-support@gsa.gov.
+Thank you,
+The cloud.gov team
+`
+
+const inviteEmailTpl = `
+<html><head><base target="_top">
```

Comment:
Where did the code come from for this email? This doesn't look fun to maintain, although emails in general are not fun to maintain and are hard to write.

I think this is why I was asking about moving this into Mailchimp. Usually those services give devs the ability to feed CSS and HTML separately and then the service will take care of inlining all the style rules which is required for an email. If this is how we usually do it, what are your thoughts on ensuring this email is maintainable in the future?